vizro-core/CHANGELOG.md

```diff
@@ -11,6 +11,28 @@ See the fragment files in the [changelog.d directory](https://github.com/mckinse
 
 <!-- scriv-insert-here -->
 
+<a id='changelog-0.2.0'></a>
+
+# 0.2.0 — 2024-04-12
+
+## Highlights ✨
+
+- Add dynamic data, which can be reloaded while the dashboard is running. An optional caching layer enables efficient refreshes with per-data source timeouts. Visit the [user guide on data](https://vizro.readthedocs.io/en/stable/pages/user-guides/data/) for more details. ([#398](https://github.com/mckinsey/vizro/pull/398))
+
+## Added
+
+- Add additional examples to docs of using custom components with custom actions. ([#392](https://github.com/mckinsey/vizro/pull/392))
+
```

Comment:
```suggestion
```

Reply:
Ah yes, we said docs changes are not worthy to be mentioned in the changelog right? Will remove!